spec/features/projects/viewing_project_spec.rb

```diff
@@ -82,8 +82,10 @@
         before :all do
           project_list_item = find('.project-list-item', match: :first)
 
-          project_list_item.find('.customize').click
+          project_list_item.find('.project-list-item-action-edit-options').click
```

Comment:
I think this file has a `pending_updates` above this test that will need to be removed for this update to run -- or is :octocat: just not showing that?